tests/functional/test_bad_url.py

```diff
@@ -8,7 +8,7 @@ def test_filenotfound_error_message(script: Any) -> None:
     # Test the error message returned when using a bad 'file:' URL.
     # make pip to fail and get an error message
     # by running "pip install -r file:nonexistent_file"
-    proc = script.pip("install", "-r", "file:unexistent_file", expect_error=True)
+    proc = script.pip("install", "-r", "file:///unexistent_file", expect_error=True)
```

Comment:
This change has been submitted separately as #12964; it's included here to prove CI isn't failing because of *this* patch.